### DISCUSSION TITLE
Not working on snapdragon 7+ gen 3

### DISCUSSION THREAD

**Discussion by CliveDandavathi** [community/community#129102] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Sir, I have a latest gaming phone ( realme gt 6t) with snapdragon 7+ gen 3 with 1.5 million antutu score and adreno 732 gpu. When I installed horizon emulator and did all the settings perfectly as mentioned in the horizon emulator home page but when I install any game in it it does not open or launch instead of downloading the game perfectly in the emulator. The game icon appears on desktop but when I click it it does not open at all. I tried prototype 2 and Assassin's creed 1 with are very low end games. Please 🙏 🙏 help and work on the emulator. I brought this phone just to play your horizon emulator and play pc games on Android

**Comment by ebndev** [comment]
Hi @CliveDandavathi, thanks for being a part of the GitHub Community!

It looks like you may have ended up here by clicking “Send feedback” on a user’s profile, is that right? 

That link is meant for users to provide feedback for Achievements, which leads to the Profile category of GitHub Discussions, a public forum.

We are working on a solution to that however, you might want to try going back to the project you were seeking support for and open an issue in the project's repository or contact them on their website / social media. There is currently no way to contact users directly on GitHub.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!
